docker/worker/worker_test.py

```diff
@@ -1644,6 +1644,37 @@ def test_invalid_prefix(self):
           source_of_truth=osv.SourceOfTruth.SOURCE_REPO,
       ).put()
 
+  def test_dont_index_too_many_git_versions(self):
+    """Test that we don't index too many versions from Git."""
+    self.source_repo.ignore_git = False
+    self.source_repo.versions_from_repo = True
+    # detect_cherrypicks should not cause result in cherrypick detection for
+    # `last_affected`, since equivalent `last_affected` across different
+    # branches likely no have relation to the actual vulnerable range.
```

Comment:
I think this is how you intended to word this?

```suggestion
    # branches likely have no relation to the actual vulnerable range.
```

Reply:
I'll remove this completely, since it's irrelevant to this test.